### PR TITLE
My Idobo Keymap

### PR DESCRIPTION
**Pull request by ifohancroft**
## Description

Just submitting my Idobo keymap for inclusion.

## Types of Changes

- [x] Keymap/layout/userspace (addition or update)
- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

File: keyboards/idobo/keymaps/ifohancroft/keymap.c [hunk 1]
```diff
@@ -0,0 +1,88 @@
+#include QMK_KEYBOARD_H
```
Comment (spidey3) [inline keyboards/idobo/keymaps/ifohancroft/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by spidey3 — CHANGES_REQUESTED** [review]

**Comment by ifohancroft** [comment]
Done

**Review by spidey3 — APPROVED** [review]

File: keyboards/idobo/keymaps/ifohancroft/config.h [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#pragma once
```
Comment (drashna) [inline keyboards/idobo/keymaps/ifohancroft/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/idobo/keymaps/ifohancroft/config.h [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+// place overrides here
+#undef USB_POLLING_INTERVAL_MS
```
Comment (drashna) [inline keyboards/idobo/keymaps/ifohancroft/config.h:4]: This shouldn't be set by default, so no need to undefine it
```suggestion
```

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/idobo/keymaps/ifohancroft/config.h (hunk 3 above)
Comment (ifohancroft) [reply keyboards/idobo/keymaps/ifohancroft/config.h:4]: Ah! Thank you! Not sure why I thought it's defined by default.

Comment (ifohancroft) [reply keyboards/idobo/keymaps/ifohancroft/config.h]: Done

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**